src/api/requestsApi.tsx

```diff
@@ -363,6 +366,12 @@ export const useUpdateStage = () => {
           </Toast>,
           { intent: "success" }
         );
+        if (request.eventTitle) {
```

Comment:
If we make the above eventTitle parameter required, then we won't need the IF wrapper here, since we know there is one